docs/sources/installation/configuration.md

```diff
@@ -355,6 +355,11 @@ Define a whitelist of allowed IP addresses or domains, with ports, to be used in
 
 Set to `true` if you host Grafana behind HTTPS. Default is `false`.
 
+
```

Comment:
Do you mind just removing this extra blank line, for consistency? It reads better without.
```suggestion
```

Reply:
Thank you and sorry for that.